gobblin-utility/src/test/java/org/apache/gobblin/util/AvroUtilsTest.java

```diff
@@ -310,4 +320,157 @@ public void testUnionWithNull() {
     String stringValue = AvroUtils.getFieldValue(record, "union.string").get().toString();
     Assert.assertEquals(stringValue, "testString");
   }
+
+
+  @Test
+  public void testDecorateSchemaWithSingleField() {
```

Comment:
Is this test needed given the other testDecorateSchema tests you have below?

Reply:
Wanted to ensure that primitive type works independently of the nested record type. Didn't want to assume that Avro treats both the same way.